swim/memberlist.go

```diff
@@ -79,11 +84,21 @@ func (m *memberlist) ComputeChecksum() {
 	startTime := time.Now()
 	m.members.Lock()
 	checksum := farm.Fingerprint32([]byte(m.GenChecksumString()))
+	oldChecksum := m.members.checksum
 	m.members.checksum = checksum
 	m.members.Unlock()
+
+	if oldChecksum != checksum {
```

Comment:
This creates a new untested code path which reduces coverage. Not saying it's the wrong approach since coverage != code quality, but if we can avoid untested code paths easily, that would be good.